AsyncPoco.Tests/AsyncPoco.Tests.csproj

```diff
@@ -80,19 +85,20 @@
     <Compile Include="MultiPocoTests.cs" />
     <Compile Include="Misc.cs" />
     <Compile Include="ColumnMapper.cs" />
-    <Compile Include="PetaTest.cs" />
     <Compile Include="Program.cs" />
-    <Compile Include="SqlBuilder.cs" />
+    <Compile Include="SqlBuilderTests.cs" />
     <Compile Include="poco.cs" />
-    <Compile Include="Tests.cs" />
+    <Compile Include="DatabaseTests.cs" />
```

Comment:
I renamed _Tests.cs_ to _DatabaseTests.cs_ to make it a little more explicit about what kinds of tests the file contains.